.github/workflows/build.yml

```diff
@@ -33,32 +41,36 @@ jobs:
             nx-${{ env.NX_BRANCH }}-
             nx-
 
-      - name: Install Dependencies
-        run: npm ci --prefer-offline --no-audit
+      - run: sudo apt-get update && sudo apt-get install -y build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev pkg-config
```

Comment:
canvas dep wasn't building properly (hopefully we can remove it with Tiptap v3 soon!)